apps/frontend/src/pages/not-found/not-found.tsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom";
+
+import styles from "./styles.module.css";
+
+const NotFound = (): JSX.Element => {
+	return (
+		<main className={styles["container"]}>
+			<h1 className={styles["title"]}>404</h1>
+			<h3 className={styles["subtitle"]}>Something went wrong</h3>
```

Comment:
404 is not a header but more like a decoration, so let's change the structure a bit
```suggestion
			<span className={styles["decoration"]}>404</span>
			<h1 className={styles["title"]}>Something went wrong</h1>
```

Reply:
changed